Resources/Locale/en-US/cluwne/cluwne.ftl

```diff
@@ -0,0 +1,2 @@
+cluwne-transform = {CAPITALIZE(THE($target))} turned into a cluwne!
+cluwne-name-prefix = Cluwne
```

Comment:
im thinking cluwnified would sound better

Reply:
yea, i will change it in the transforms, i can probably drop the prefix part since cluwne simply replaces their whole name and they become a cluwne now.

Reply:
cluwne is now cluwnified (name) instead of just being called cluwne